### PR TITLE
Support assignments

### PR DESCRIPTION
**Pull request by szok**
- Added support and licence to back office asset
- Added support and licence to data center asset

Connected to #1699

### CODE REVIEW

File: src/ralph/back_office/admin.py [hunk 1]
```diff
@@ -1,15 +1,48 @@
...
+
+
+class BackOfficeAssetSupport(RalphDetailViewAdmin):
+    icon = 'bookmark'
+    name = 'bo_asset_support'
+    label = _('Supports')
+    url_name = 'back_office_asset_support'
+
+    class BackOfficeAssetSupportInline(RalphTabularInline):
+        model = Support.base_objects.through
```
Comment (mkurek) [inline src/ralph/back_office/admin.py]: maybe it'll be better to use `BackOfficeAsset.supports.related.through` here? It's the same table - the only change it that in your example we're going to it from support (touching base object) and in mine there is straightforward way from BO asset to supports.

File: src/ralph/back_office/admin.py [hunk 2]
```diff
@@ -1,15 +1,48 @@
...
+    icon = 'bookmark'
+    name = 'bo_asset_licence'
+    label = _('Licence')
+    url_name = 'back_office_asset_licence'
+
+    class BackOfficeAssetLicenceInline(RalphTabularInline):
+        model = BaseObjectLicence
+        raw_id_fields = ('licence',)
+        extra = 1
+        verbose_name = _('Support')
```
Comment (mkurek) [inline src/ralph/back_office/admin.py]: licence

File: src/ralph/back_office/admin.py [hunk 3]
```diff
@@ -1,15 +1,48 @@
...
+        raw_id_fields = ('support',)
+        extra = 1
+        verbose_name = _('Support')
+
+    inlines = [BackOfficeAssetSupportInline]
+
+
+class BackOfficeAssetLicence(RalphDetailViewAdmin):
+
+    icon = 'bookmark'
```
Comment (mkurek) [inline src/ralph/back_office/admin.py]: use different icon for supports and licences.

**Comment by mkurek** [comment]
Apply comments from BO to DC too.

**Comment by ar4s** [comment]
:ok:

File: src/ralph/data_center/admin.py [hunk 4]
```diff
@@ -58,6 +58,35 @@ class NetworkView(RalphDetailViewAdmin):
     inlines = [NetworkInline]
 
 
+class DataCenterAssetLicence(RalphDetailViewAdmin):
```
Comment (mkurek) [inline src/ralph/data_center/admin.py]: use the same order in DC and BO admin modules (here is `DataCenterAssetLicence` then `DataCenterAssetSupport`, in BO it's reversed).

**Comment by mkurek** [comment]
@szok fix my small notice, make it mergable and it's :ok: for me

**Pull request merged**